tests/misc_env/install_prereq.py

```diff
@@ -133,15 +130,23 @@ def install_prereq(
             ceph.exec_command(cmd="sudo systemctl restart NetworkManager.service")
 
         if not skip_subscription:
-            setup_subscription_manager(ceph, is_production, cloud_type)
+            if not setup_subscription_manager(ceph, "cdn"):
```

Comment:
I think we should not propagate to failback system on the upgrade scenario or when CDN and stage subscription are must required.

Reply:
This change is only for RHEL repositories. For upgrade scenarios, we need ceph repositories which are enabled in the bootstrap step.

For providing option to subscribe to CDN or Stage, needs changes in run.py which will be added in next phase of changes.